pandas/tests/indexes/datetimes/test_date_range.py

```diff
@@ -399,7 +400,6 @@ def test_daterange_bug_456(self):
         # GH #456
         rng1 = bdate_range('12/5/2011', '12/5/2011')
         rng2 = bdate_range('12/2/2011', '12/5/2011')
-        rng2.freq = BDay()
```

Comment:
This is redundant, as `bdate_range` should already set this as the frequency.  I couldn't find a test that verified this though, so I added a check to a test a few lines above.  Same deal with the `CDay()` changes below.